aws/sdk/integration-tests/aws-smithy-runtime-test/src/endpoints.rs

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use aws_smithy_http::body::SdkBody;
+use aws_smithy_runtime::{BoxError, EndpointOrchestrator};
+use aws_smithy_runtime_api::config_bag::ConfigBag;
+use aws_smithy_runtime_api::runtime_plugin::RuntimePlugin;
+
+#[derive(Debug)]
+pub struct GetObjectEndpointOrc {}
+
+impl GetObjectEndpointOrc {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+impl RuntimePlugin for GetObjectEndpointOrc {
```

Comment:
Haven't fully decided yet but we may need to define a separate struct `GetObjectEndpointPlugin` and `impl RuntimePlugin` on that.